wireapp/wire-android-reloaded
# chore: add basic config for coverage

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  
- The **PR Description**
----

# What's new in this PR?

### Issues

As I was looking at the configuration on `quality.gradle.kts` we already had configured JaCoCo, also [CodeCov](https://app.codecov.io/gh/wireapp/wire-android-reloaded/commits?page=1) was also configured for the project, so I just did minor config changes, that enabled the coverage report to be displayed on every PR.

I had to enable that our unit-tests action, also were triggered on `develop` branch, so we can have a point of comparison

### Testing

#### How to Test

You can go to CodeCov app and see the reports uploaded, otherwise you can also check the CodeCov commenting the PR:

- https://app.codecov.io/gh/wireapp/wire-android-reloaded/commits?page=1
- https://github.com/wireapp/wire-android-reloaded/pull/510#issuecomment-1099108405

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
